lib/SIL/IR/SILInstructions.cpp

```diff
@@ -1371,7 +1362,15 @@ unsigned AssignOrInitInst::getNumInitializedProperties() const {
 ArrayRef<VarDecl *> AssignOrInitInst::getInitializedProperties() const {
   if (auto *accessor = getReferencedInitAccessor())
     return accessor->getInitializedProperties();
-  return {};
+  else {
+    // Dealing wtih an init accessor thunk, only initializes backing property
+    // FIXME: Tempory solution below, cannot allocate each time this function is
+    // called
```

Comment:
The long-term plan is to store initialized/accessed properties in the instruction itself, let's address this termporarily by supplying this method with a callback parameter that is going to be called for every property i.e. `llvm::function_ref<void(VarDecl *)> callback` this way we can avoid allocating a copy in the context every time.